homeassistant/components/nzbget/__init__.py

```diff
@@ -2,7 +2,6 @@
 from datetime import timedelta
 import logging
 
-import pynzbgetapi
```

Comment:
Please revert this change. We want 3rd party imports at the top of the module.

Reply:
@MartinHjelmare  -- doh. Here: https://developers.home-assistant.io/docs/en/0.81.0/creating_component_code_review.html.  It says: "Requirements should only be imported inside functions. This is necessary because requirements are installed on the fly."  

Should this documentation be updated?

Reply:
That page is for an old version of the docs. This is the current version:
https://developers.home-assistant.io/docs/en/creating_component_code_review.html